cli/module_registry.go

```diff
@@ -209,21 +200,26 @@ func UploadModuleAction(c *cli.Context) error {
 				"if you want to upload a version without a meta.json, you must supply a module name and namespace (or module name and org-id)",
 			)
 		}
-		moduleID, err = updateManifestModuleIDWithArgs(c, client, nameArg, publicNamespaceArg, orgIDArg)
-		if err != nil {
-			return err
-		}
+		// get from the BE?
```

Comment:
```suggestion
		moduleID, err = validateModuleID(c, client, nameArg, publicNamespaceArg, orgIDArg)
```

but what I just suggested is a bit hacky & relies on the current behavior of `validateModuleID`

Could also probably just use a few if/else statements to create the moduleid

Reply:
> [nit] but what I just suggested is a bit hacky & relies on the current behavior of validateModuleID

not sure what you are referring to here as your suggestion

Reply:
> not sure what you are referring to here as your suggestion

I am referring to the `Suggested change` in my comment

Reply:
Sorry for the confusion! probably could have written that better!

Reply:
ahah lol i missed that box !! yess you are right forgot to add there but yes can be refactored. will do that now